lisa/sut_orchestrator/hyperv/platform_.py

```diff
@@ -334,3 +378,255 @@ def _delete_node(node_ctx: NodeContext, wait_delete: bool) -> None:
                 for node in environment.nodes.list()
             ]
         )
+
+
+class HyperVDevicePoolImpl(BaseDevicePoolImpl):
+    def __init__(
+        self,
+        node: RemoteNode,
+        assignable_device_script: PurePath,
+        runbook: HypervPlatformSchema,
+    ) -> None:
+        # Device Passthrough configs
+        # Mapping of Host Device Passthrough
+        self.AVAILABLE_HOST_DEVICES: Dict[
```

Comment:
SCREAMING_SNAKE_CASE is for static/global variables. Member variables should use snake_case.

Reply:
@cwize1 

Done